stub-environment/roles/BCPC-Hadoop-Head-HBase.json

```diff
@@ -3,7 +3,9 @@
   "json_class": "Chef::Role",
   "run_list": [
     "role[Basic]",
+    "recipe[bcpc-hadoop::phoenixqs_hbase_site]",
```

Comment:
Can we include this as part of ``recipe[bcpc-hadoop::phoenix_query_server]`` as we do in [hbase_master.rb](https://github.com/pu239ppy/chef-bach/blob/f5149e814fd83cefda319de5adf45d68fe0ea1fc/cookbooks/bcpc-hadoop/recipes/hbase_master.rb#L3)